reference/storage-class-parameters.md

```diff
@@ -16,23 +16,17 @@ The setting is supported only when using "nvmf" protocol.
 
 ## "local"
 
-It is a boolean-type flag with the default value of false. All following values are interpreted as "true" and anything else as "false": 'y', 'Y', 'yes', 'Yes', 'YES', 'true', 'True', 'TRUE', 'on', 'On', 'ON'. The flag controls scheduling of nexus and replicas to storage nodes in the cluster. Mayastor always tries to schedule nexus and one of the replicas on the same node as the application that is using the volume for two reasons:
+A flag of the type Boolean, with the default value of true.  The flag controls the scheduling behaviour of nexus and replicas to storage nodes in the cluster.
 
-1. minimal number of network hops in the IO path and
-2. simplified failure recovery if the application and nexus run on the same node.
+All the following values are interpreted as "true" and anything else as "false": 'y', 'Y', 'yes', 'Yes', 'YES', 'true', 'True', 'TRUE', 'on', 'On', 'ON'. 
 
-The only difference depending on the flag's value is what happens when it isn't possible to schedule the application and nexus on the same node. If the value of "local" is true, then the application will become unscheduleable and will not start. If "local" is false, then the application will run regardless but on a different node than the nexus.
-
-{% hint style="info" %}
-Note that in order to enable the locality feature, the `volumeBindingMode` in storage class must be set to `WaitForFirstConsumer`.
+{% hint style="warning" %}
+This value must be set to true for correct operation of Mayastor provisioning and publishing.  It is recommended that  the `volumeBindingMode` in storage class be set to `WaitForFirstConsumer`.  This limitation will be removed in a future release
```

Comment:
Right so now we're saying that it must be set to true, so we don't need to explain what happens with false, and we'll "fix" this anyway on the next release?

Reply:
That's might intention, based on conversations I believe we've had.  Do you concur?

Reply:
We've had an issue raised recently where someone was trying to use local false; they expected to stand a volume on a non-engine node which currently is not supported even with local false.

> Dr. Harris, do you concur? 
I do :)

Reply:
Then we have consensus.  :-)